AY2122S2-CS2103T-T09-1/tp
# Update tests

**Summary of Changes:**

Uncomment with no changes:
- `HelpCommandTest`
- `ExitCommmandTest`
- `ClearCommandTest`
- `CommandResultTest`
- `AddCommandIntegrationTest`
- `ArgumentTokenizerTest`

Uncomment with some modification:
- `EditCommandTest`
  - Replace `getFilteredTaskList()` with `getSortedTaskList()`
- `EditTaskDescriptorTest`
   - Add check for priority in `equals()`
- `EditTaskDescriptionBuilder`
    - Add method `withPriority`
- `AddCommandTest`
   - Update `ModelStub` with `getSortedTaskList()`, `updateSortedTaskList()`, `getTagList()`
- `AddCommandParserTest`
   - Update tests to include Priority
- `EditCommandParserTest`
   - Update tests to include Priority
- `ParserUtilTest`
   - Update test to include Priority
- `LogicManagerTest`
   - Refactor tests from `AddressBook` to `Harmonia`
 
Add tests for list tags.
